### PR TITLE
WIP: feat: Remove deprecated request package in favor of axios

### PR DESCRIPTION
**Pull request by dunklesToast**
As request was deprecated at the begin of this year I thought it would be good if it would get replaced.

The functionality is already ported to axios but I have some issues with the tests. Would be great if somebody could assist me there.

### CODE REVIEW

File: scripts/install.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ var fs = require('fs'),
   sass = require('../lib/extensions'),
   request = require('request'),
```
Comment (stephanvierkant) [inline scripts/install.js:10]: Should be removed?

**Comment by stephanvierkant** [comment]
You've added yarn.lock (by accident?).

**Comment by dunklesToast** [comment]
I added yarn.lock because I use yarn as a package manager. I will remove it once the PR is ready to merge.

File: scripts/install.js (hunk 1 above)
Comment (dunklesToast) [reply scripts/install.js:10]: Yes. Missed that.

**Comment by easingthemes** [comment]
Check https://github.com/sass/node-sass/issues/2851, `request` replacement. is in consideration, and final plugin is not decided yet.

**Comment by dunklesToast** [comment]
Okay it seems that make-fetch-happen should be used. If that decision is final I'd like to create a PR for that.

**Pull request closed without merging**